src/Session.dfy

```diff
@@ -89,12 +89,12 @@ module Session {
       case Error(r) =>
         r != AWS_OP_SUCCESS
       case Done => true
-      // decryption
+                   // decryption
       case ReadHeader => true
       case UnwrapKey => true
       case DecryptBody => true
       case CheckTrailer => true
-      // encryption
+                           // encryption
```

Comment:
2 spaces for things like `/**` but lots for `//`?

Reply:
The decision was not taken for the comment type, but for where it applies.
However, I see that this is really not what people write comments for. Everyone wants a comment before a case to be aligned with the case itself. Hence, let me change the formatter so that it's the case (pun intended -) )